### ISSUE TITLE
SIRI EstimatedTimetable et filtre par LineRef

### ISSUE THREAD

**Issue by AntoineAugusti** [etalab/transport-normes#38]
Question concernant le profil SIRI, pour `GetEstimatedTimetable`. Quelle est la requête appropriée pour filtrer selon une seule ligne ? Faut-il avoir `Lines > LineDirection > LineRef` ou uniquement `Lines > LineRef` ?

[Sur le site actuellement](https://normes.transport.data.gouv.fr/normes/siri/profil-france/#estimated-timetable), il me semble qu'on indique uniquement `Lines > LineRef`.

![image](https://github.com/etalab/transport-normes/assets/295709/6205ac87-534b-4be3-98ea-b134d9345bf6)

[Côté EnRoute](https://www.postman.com/enroute-mobi/workspace/enroute-s-public-workspace/request/9203997-719abed0-74a7-4ace-a594-190365518331) cc @albanpeignier, la requête fait apparaitre ceci.

```
<sw:GetEstimatedTimetable xmlns:sw="http://wsdl.siri.org.uk" xmlns:siri="http://www.siri.org.uk/siri">
    <ServiceRequestInfo>
        <siri:RequestTimestamp>{{current_time}}</siri:RequestTimestamp>
        <siri:RequestorRef>{{partner_token}}</siri:RequestorRef>
        <siri:MessageIdentifier>Test:Message::{{$randomUUID}}:LOC</siri:MessageIdentifier>
    </ServiceRequestInfo>
    <Request version="2.0:FR-IDF-2.4">
        <siri:RequestTimestamp>{{current_time}}</siri:RequestTimestamp>
        <siri:MessageIdentifier>Test:Message::{{$randomUUID}}:LOC</siri:MessageIdentifier>
        <siri:Lines>
          <siri:LineRef>{{line_identifier}}</siri:LineRef>
      </siri:Lines>
  </Request>
  <RequestExtension />
</sw:GetEstimatedTimetable>
```

[Côté CEN 🇪🇺](https://github.com/SIRI-CEN/SIRI/blob/aca7fa73f8e544f7298b7cb62a823e8dc520ee2a/examples/siri_exm_ET/ext_estimatedTimetable_subscriptionRequest.xml#L18-L23), il y a un wrap `LineDirection` qui est présent.

cc @Aurige

**Comment by albanpeignier** [comment]
En effet, la [documentation postman d'Ara SaaS](https://documenter.getpostman.com/view/9203997/SzmY7LiV) fait référence à une structure invalide:

```
➜ be exe/siri-xsd-validate test.xml
Validate test.xml
Inspect test.xml: 1 error(s)
test.xml:16:0: ERROR: Element '{http://www.siri.org.uk/siri}LineRef': This element is not expected. Expected is ( {http://www.siri.org.uk/siri}LineDirection ).
```

L'erreur vient d'un vieux document:

![image](https://github.com/etalab/transport-normes/assets/24570/316f25a7-8e4d-425e-9248-f97a0b4b41df)
![image](https://github.com/etalab/transport-normes/assets/24570/4438e92e-b205-43b0-9d8c-f4ab9ba9bf7f)

où il fallait interpréter la structure du tableau pour comprendre que la forme correcte fait intervenir un `LineDirection` intermédiaire.

A noter qu'Ara gère les deux formes, dont la forme valide:

```xml
<?xml version="1.0" encoding="utf-8"?>
<soap:Envelope xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/">
  <soap:Header/>
  <soap:Body>
    <sw:GetEstimatedTimetable xmlns:sw="http://wsdl.siri.org.uk" xmlns:siri="http://www.siri.org.uk/siri">
      <ServiceRequestInfo>
        <siri:RequestTimestamp>2030-01-01T12:00:00.000Z</siri:RequestTimestamp>
        <siri:RequestorRef>enRoute</siri:RequestorRef>
        <siri:MessageIdentifier>3a2ac89a-6376-11ee-b1b8-4bee866f6f22</siri:MessageIdentifier>
      </ServiceRequestInfo>
      <Request>
        <siri:RequestTimestamp>2030-01-01T12:00:00.000Z</siri:RequestTimestamp>
        <siri:MessageIdentifier>3a2ac89a-6376-11ee-b1b8-4bee866f6f22</siri:MessageIdentifier>

        <siri:Lines>
          <siri:LineDirection>
            <siri:LineRef>42</siri:LineRef>
          </siri:LineDirection>
        </siri:Lines>
      </Request>
      <RequestExtension />
    </sw:GetEstimatedTimetable>
  </soap:Body>
</soap:Envelope>
```

Nous allons mettre à jour notre documentation.

**Comment by thbar** [comment]
Merci @albanpeignier! On a fait évolué notre requêteur SIRI pour utiliser `Lines > LineDirection > LineRef` (forme "correcte").

Du coup il va falloir faire évoluer le markdown du profil si je comprends bien ?

Si c'est bien le cas, il faut voir si on finalise d'abord #32 ou bien si la correction est faite après (j'aimerais autant ne pas apporter de modifications sémantiques sur #32 car c'est plus simple à tracer après).

**Comment by albanpeignier** [comment]
> Du coup il va falloir faire évoluer le markdown du profil si je comprends bien ?

Oui. Le markdown du Profil SIRI fait la même simplification (invalide) que notre documentation. Peut être juste en précisant la chose dans la description de `Lines`:

> Liste de `LineDirections` définissant les lignes contenant les courses pour lesquelles on souhaite des informations.
